algos/src/sample_rate_conversion.rst

```diff
@@ -75,139 +75,138 @@ scripts print intermediate information about progress).
-int32 coefficient set. The testbench and FW build with xtensa compiler
-defaults to 32 bit coefficients. The gcc build build for firmware
-uses 16 bit coefficients. The scripts those were use to generated them
-are src_tiny_int16.m and src_std_int32.m.
+The type of SRC component used depends on whether you are building a "tiny"
+int16 or "std" int32 coefficient set. The testbench and FW build with xtensa
+compiler defaults to 32 bit coefficients. The gcc build for firmware uses 16
+bit coefficients. The scripts those were use to generated them are
+src_tiny_int16.m and src_std_int32.m.
 
 Exclusion of non-needed conversions
 ===================================
 
-If from previous call conversion set there would be no need for
-converting from 32 to 44.1 kHz it can be achieved by adding a matrix
-with zero in the place of the non-wanted conversion. This will help
-save memory that is needed to store the conversion coefficients.
+If previous call conversion sets had no need for converting from 32 to 44.1
+kHz, it can be achieved by adding a matrix with zero in the place of the
+non-wanted conversion. This will help save memory that is needed to store
+the conversion coefficients.
 
 .. code-block:: octave
 
    >> src_generate([32e3 44.1e3 48e3],[44.1e3 48e3],[0 1; 1 1; 1 1])
 
-In the script output the removed conversion is marked with an ‘x’ and
-correspond filters are not calculated.
+In the script output, the removed conversion is marked with an ‘x’ and
+corresponding filters are not calculated.
 
 Adjustment of SRC filter specification
 ======================================
 
 The default conversions are tuned with the stopband specification to
 provide min -80 dBFs THD+N performance. The requested stop-band
-attenuation has been chosen that the THD+N criteria is met in the
+attenuation has been chosen such that the THD+N criteria is met in the
 worst-case modes.
 
-The bandwidth is about 20 kHz bandwidth for 44.1 kHz and 48 kHz sample
-rates. The bandwidth scales with minimum sample rate of
-conversion. However for rates higher than 88.1 kHz the bandwidth is
+The bandwidth is about 20 kHz for 44.1 kHz and 48 kHz sample
+rates. The bandwidth scales with a minimum sample rate of
+conversion. However, for rates higher than 88.1 kHz, the bandwidth is
 kept as about 30 kHz to provide a measurable band extension but
-however not stretch it near Nyquist Fs/2 as for lower sample rates.
+not stretch it near Nyquist Fs/2 as for lower sample rates.
 
-The transition band starts from filter pass-band bandwidth and ends at
-stop-band start. It is e.g. from 20 kHz to Nyquist rate Fs/2. The
-transition band is don’t care region for filter-design but with the
-used filter design method it connects end of pass-band to start of
-stop-band with near constant dB/log frequency line them.
+The transition band starts at the filter pass-band bandwidth and ends at
+the stop-band start. It is e.g. from 20 kHz to Nyquist rate Fs/2. The
```

Comment:
I do not understand the use of e.g. in this sentence.

Reply:
This is good!